typescript/infra/scripts/agent-utils.ts

```diff
@@ -617,15 +617,28 @@ function getInfraLandfillPath(environment: DeployEnvironment, module: Modules) {
   return path.join(getModuleDirectory(environment, module), 'addresses.json');
 }
 
-export function getAddresses(environment: DeployEnvironment, module: Modules) {
+export function getAddresses(
+  environment: DeployEnvironment,
+  module: Modules,
+  chains: ChainName[] = [],
```

Comment:
nit: may be more natural to have this be `ChainName[] | undefined` instead of treating `[]` as a no filter